powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -102,6 +102,18 @@ public void setServiceBaseUrl(String serviceBaseUrl) {
     }
 
     /**
+     * Returns service information
+     *
+     * @return True if service is running.
+     */
+    public boolean getServiceStatus() throws PushServerClientException {
+        TypeReference<ServiceStatusResponse> typeReference = new TypeReference<ServiceStatusResponse>() {
+        };
+        ObjectResponse<?> response = getObjectImpl("/push/service/status", null, typeReference);
+        return response.getStatus().equals(Response.Status.OK);
```

Comment:
Return the whole `ServiceStatusResponse` here - there are more information that may be useful to the developer in the response object...